sdp-utils.c

```diff
@@ -1282,7 +1282,7 @@ janus_sdp *janus_sdp_generate_offer(const char *name, const char *address, ...)
 				char *extmap = g_hash_table_lookup(audio_extids, iter->data);
 				if(extmap != NULL) {
 					janus_sdp_attribute *a = janus_sdp_attribute_create("extmap",
-						"%d %s\r\n", GPOINTER_TO_INT(iter->data), extmap);
```

Comment:
Correct me if this was supposed to be like this, but it does not appear in any other `janus_sdp_attribute_create` calls. I removed it because it messes up extmap value matching (I could've just trimmed extmap values but this seemed to be more correct).